src/lib/purge_cache.ts

```diff
@@ -0,0 +1,46 @@
+import { env } from 'process'
+
+interface PurgeCacheOptions {
+  apiURL?: string
+  siteID?: string
+  tags: string[]
```

Comment:
The API can take all of these arguments (except `paths`): https://www.notion.so/netlify/Cache-Purge-API-12b8eb7359c549a4aad56d528f19feb0?pvs=4#46c26a47bd3a4689bb38dcd97c7b6eae